recipe/yum_requirements.txt

```diff
@@ -0,0 +1,5 @@
+java-1.8.0-openjdk-devel
+unzip
+zip
```

Comment:
Was proposing we add `zip` and `unzip` based on a user request in PR ( https://github.com/conda-forge/docker-images/pull/20 ). Haven't gotten any feedback on it though. Should we be adding those?